### PR TITLE
[KEYCLOAK-9128] Vendor update as of 2019-01-01

### PR DESCRIPTION
**Pull request by fredbi**
Signed-off-by: Frederic BIDON <frederic@oneconcern.com>

### CODE REVIEW

**Comment by fredbi** [comment]
@abstractj the vendor update caused tests to fail - the test update just reflects the change in the updated dependencies.

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,40 @@
+module github.com/fredbi/keycloak-gatekeeper
```
Comment (abstractj) [inline go.mod]: @fredbi could you please replace your username by "keycloak"? Other than that we're good to go.

Comment (fredbi) [reply go.mod]: @abstractj ah yes, that's this automatic go mod update... Fixing it thanks

**Comment by fredbi** [comment]
@abstract I am removing the go.mod specification for the moment, as it is still difficult for CI to figure out some other broke dependencies. There are some other dependency fixes to be carried out to be able to properly build with go modules. I'd rather deferring this to another PR..

**Comment by abstractj** [comment]
Superseded by https://github.com/keycloak/keycloak-gatekeeper/pull/466 which is just a rebase of this PR.

Thank you.

**Pull request closed without merging**